src/client/Microsoft.Identity.Client/Platforms/Android/Broker/AndroidBrokerFactory.cs

```diff
@@ -52,6 +53,7 @@ private static async Task<IBroker> GetInstalledBrokerAsync(CoreUIParent uIParent
                         var broker = new AndroidAccountManagerBroker(uIParent, logger);
                         await broker.InitiateBrokerHandshakeAsync().ConfigureAwait(false);
                         s_installedBroker = BrokerType.AccountManager;
+                        logger.Info("Account manager broker is available and will be used.");
```

Comment:
logger.Error

Reply:
This is not a log error. The error message for this is printed on line 48